### PR TITLE
To absolute path

### PR DESCRIPTION
**Pull request by wli3**
https://github.com/dotnet/xliff-tasks/issues/29

(Mutex will be a separate PR)

### CODE REVIEW

**Comment by wli3** [comment]
the test won't work on unix

File: src/XliffTasks.Tests/ResxDocumentTests.cs [hunk 1]
```diff
@@ -45,7 +45,40 @@ public void BasicLoadAndTranslate()
...
+</root>";
+
+            var document = new ResxDocument();
+            var writer = new StringWriter();
+            document.Load(new StringReader(source));
+            document.RewriteRelativePathsToAbsolute(
+                @"E:\sourceFolder\Resources.resx");
+            document.Save(writer);
+
+            AssertHelper.AssertWithoutLineEndingDifference(expectedTranslation, writer.ToString());
```
Comment (nguerrera) [inline src/XliffTasks.Tests/ResxDocumentTests.cs]: Can we use AssertEx.EqualIgnoringLineEndings like I suggested on the original PR?

File: src/XliffTasks/Model/ResxDocument.cs [hunk 2]
```diff
@@ -52,5 +54,23 @@ protected override IEnumerable<TranslatableNode> GetTranslatableNodes()
...
         }
+
+        public override void RewriteRelativePathsToAbsolute(string sourceFullPath)
+        {
+            foreach (var node in Document.Descendants("data"))
+            {
+                if (node.Attribute("type")?.Value == "System.Resources.ResXFileRef, System.Windows.Forms")
+                {
+                    var valueNodeOfFileRef = node.Element("value");
+                    var splittedRelativePathAndSerializedType = valueNodeOfFileRef.Value.Split(';');
```
Comment (nguerrera) [inline src/XliffTasks/Model/ResxDocument.cs]: splitted is not a word. `splitRelativePathAndSerializedType`.

File: src/XliffTasks.Tests/ResxDocumentTests.cs (hunk 1 above)
Comment (wli3) [reply src/XliffTasks.Tests/ResxDocumentTests.cs]: Sorry, I missed this change. I will add it

File: src/XliffTasks/Model/ResxDocument.cs (hunk 2 above)
Comment (wli3) [reply src/XliffTasks/Model/ResxDocument.cs]: fixed

File: src/XliffTasks.Tests/ResxDocumentTests.cs (hunk 1 above)
Comment (wli3) [reply src/XliffTasks.Tests/ResxDocumentTests.cs]: fixed

**Comment by wli3** [comment]
Thanks

**Pull request merged**